ubuntudesign/vanilla-docs-theme
# Restructure files and mixins

I've restructured the files and code in the following ways:

- Only import the parts we need from vanilla-framework, rather than the whole thing
- Import vanilla dependencies "just in time" - patterns will import the patterns they depend on directly
- Rename "theme_" patterns to "patterns_" to follow the vanilla standard
- Namespace all pattern mixins with "docs-" (like vanilla's "vf-")
- Remove from the demo page elements which aren't part of this theme

QA
--

First, checkout the `v1.2.0` branch, build the theme and open `demo/index.html`. Keep that open in one tab.

Now checkout this branch, build the theme and open `demo/index.html` in a different tab.

The two demos should look identical, apart from that the new one won't have:                                      

- The versions section in the left nav (a new versions pattern is coming imminently)
- The "Jump to content" link in the header on mobile view (this pattern is not part of the theme)
